python/create_service_diagnostics_bundle.sh

```diff
@@ -5,11 +5,10 @@ set -eu -o pipefail
 readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly DCOS_SERVICE_DIAGNOSTICS_SCRIPT_PATH="dcos-sdk-service-diagnostics/python"
 
-#ToDo: the image version should be externally managed outside this file.
-readonly VERSION='v0.4.0'
+readonly BUILD_IMAGE_VERSION=${BUILD_IMAGE_VERSION:-$(<VERSION)}
```

Comment:
Can you please explain how is this going to work on an end-user's machine?

Reply:
Right question and 
@mpereira has the same right question. I think, we can replace ${BUILD_IMAGE_VERSION:-$(<VERSION)}` part with concrete version before upload the script to S3 for GA. Making it `readonly BUILD_IMAGE_VERSION=v0.4.1`. I will update CONTRIBUTING.md appropriately.

Reply:
If that's the approach taken, let's keep the environment variable name as just `VERSION`.

Reply:
`VERSION` is too ambiguous name, on my taste. Is it version of the script, or only docker image version. BUILD_IMAGE_VERSION is more concrete.

Reply:
From the user perspective, there should only be one version: the DC/OS Service Diagnostics version. IMO, the current _implementation detail_ of having a versioned script that runs a versioned Docker image shouldn't be made apparent in the user-facing software (the script).

Also, from a user perspective, the script should never have a different version than a Docker image. They're one: the DC/OS Service Diagnostics version.

Reply:
PTAL. Also updated instruction in CONTRIBUTING.md.